rskj-core/src/test/java/co/rsk/peg/BridgeTest.java

```diff
@@ -469,8 +471,8 @@ public void registerFastBridgeBtcTransaction_after_RSKIP176_activation_generic_e
             true
         );
         byte[] result = bridge.execute(data);
-        
-        Assert.assertEquals(BridgeSupport.FAST_BRIDGE_GENERIC_ERROR, 
+
+        Assert.assertEquals(BridgeSupport.FAST_BRIDGE_GENERIC_ERROR,
```

Comment:
Maybe we can do a static import of `Assert` to improve readability? I've seen it in some of Angel's tests here and there

Reply:
Will do!

Reply:
Done!